OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -20,9 +20,15 @@ public class WithInfantryBodyInfo : ITraitInfo, IQuantizeBodyOrientationInfo, IR
 	{
 		public readonly int MinIdleWaitTicks = 30;
 		public readonly int MaxIdleWaitTicks = 110;
-		public readonly string MoveSequence = "run";
-		public readonly string AttackSequence = "shoot";
+
+		[SequenceReference] public readonly string MoveSequence = "run";
+
+		[SequenceReference] public readonly string AttackSequence;
```

Comment:
@Mailaender's comment just above shows that in this particular case "shoot" is actor-specific.
Also you can refer to repairing/reloading to see "fix" being hardcoded as a default, which is pretty mod-specific.